internal/pkg/gateway/apiutils.go

```diff
@@ -45,7 +45,7 @@ func getChannelAndChaincodeFromSignedProposal(signedProposal *peer.SignedProposa
 	return channelHeader.ChannelId, spec.ChaincodeSpec.ChaincodeId.Name, len(payload.TransientMap) > 0, nil
 }
 
-func rpcError(code codes.Code, message string, details ...proto.Message) error {
+func newRpcError(code codes.Code, message string, details ...proto.Message) error {
```

Comment:
Out of curiosity, why the change of function name?

Reply:
I introduced a `toRpcError` function and just wanted the names to be a little more distinct to avoid confusion